run_dir/static/js/fc_trend_plot.js

```diff
@@ -78,6 +99,14 @@ function make_plot(key, name, display_by, filter_inst_type, filter_inst, color_t
         }
     };
 
+    function addBP() {
+        toplot.tooltip.pointFormat = '{series.name} : <b>{point.y}</b><br />Mbp: <b>{point.bp_yield:,.0f}</b>';
+    }
+
+    if (display_by == "flowcell") {
+        addBP();
+    }
+
```

Comment:
If this is the only place addBP is used why can't this be
```suggestion
    if (display_by == "flowcell") {
        toplot.tooltip.pointFormat = '{series.name} : <b>{point.y}</b><br />Mbp: <b>{point.bp_yield:,.0f}</b>';
    }

```

Reply:
Also from old code 👍🏻